src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -99,5 +99,13 @@ public override bool IsExplicitMatch(ITest test)
 
         //    return false;
         //}	
+
+        public override TNode AddToXml(TNode parentNode, bool recursive)
+        {
+            TNode result = parentNode.AddElement("not");
+            if (recursive)
+                BaseFilter.AddToXml(result, true);
+            return result;
+        }
     }
```

Comment:
This class doesn't need the ElementName override because it does not derive from CompositeFilter. Like CompositeFilter, it overrides AddToXml instead because it derives directly from TestFilter.